README.md

```diff
@@ -59,3 +59,34 @@ IMPORTANT! The `importer` crate's dependency on `osm2streets` must be kept in
 sync with the git version used by ATIP's [route info
 crate](https://github.com/acteng/atip/tree/map_model/route_info). Otherwise,
 the binary file format may be incompatible. Use `cargo update -p osm2streets`.
+
+## POIs (points of interest)
+
+ATIP loads extra layers showing key trip generators. These layers are
+England-wide, rather than being split into a file per area, because they're
+being used on the country-wide scheme browse page. Each layer is a single
+[PMTiles](https://protomaps.com/docs/pmtiles/) file.
```

Comment:
The pmtiles docs are worth skimming through for more background. In short, we're using them as a much more performant variation of a .geojson file. We don't want to load the entire England-wide dataset, just the areas that the user zooms in on. Vector tiles are the general technology that achieve this. Serving vector tiles is kind of tedious, and pmtiles are a newer approach that's simple and fast. They're packed into a single file, and the browser uses HTTP range requests to grab only the necessary parts.